### PR TITLE
chore(ci,docs): disable ToC in perf gh pages.

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind cleanup

**Any specific area of the project related to this PR?**

/area CI

**Does this PR require a change in the driver versions?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: .github/workflows/pages.yml [hunk 1]
```diff
@@ -106,11 +106,11 @@ jobs:
         run: |
           python3 .github/generate_inline_svg_md.py trace_tests.svg
           mv out.md docs/unit_tests.md
-          rm -rf out.md
+          sed -i '1s/^/---\nhide:\n- toc\n---\n\n/' docs/unit_tests.md
```
Comment (FedeDP) [inline .github/workflows/pages.yml:109]: Same as for kernel testing matrixes.

File: docs/perf.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # Home of Falco Perf Monitoring
 
-Our CI is capable of continuosuly benchmarking performance of our userspace code.  
+Our CI is capable of continuously benchmarking performance of our userspace code.  
 Every PR will have a comment with the perf diff from master for multiple aspects, while on master the flamegraph are pushed to this github pages.
 
 Navigate to the perf reports on the left, or click these links:
+
```
Comment (FedeDP) [inline docs/perf.md:7]: Properly render bullet list.

**Review by jasondellaluce — APPROVED** [review]

File: .github/generate_inline_svg_md.py
Comment (FedeDP) [inline .github/generate_inline_svg_md.py:1]: Locally testing with `mkdocs build` and `mkdocs serve`; it seems like it is able to create a bundle `site/` with proper link to svg images; we don't need to inline them anymore.

File: .github/workflows/pages.yml [hunk 3]
```diff
@@ -93,24 +93,24 @@ jobs:
...
         run: |
-          python3 .github/generate_inline_svg_md.py trace_tests.svg
-          mv out.md docs/unit_tests.md
-          rm -rf out.md
-          python3 .github/generate_inline_svg_md.py trace_scap.svg
-          mv out.md docs/scap_file.md
-          rm -rf out.md    
-          
+          mv trace_tests.svg docs/trace_tests.svg
+          echo '![unit tests](./trace_tests.svg)' > docs/unit_tests.md
```
Comment (FedeDP) [inline .github/workflows/pages.yml:104]: Create the `unit_tests.md` file that references the svg image. Same below for scap file.

**Comment by FedeDP** [comment]
/hold
while i test other improvements.

**Review by Andreagit97 — DISMISSED** [review]
/approve

File: .github/workflows/pages.yml [hunk 4]
```diff
@@ -93,24 +93,24 @@ jobs:
...
         run: |
-          python3 .github/generate_inline_svg_md.py trace_tests.svg
-          mv out.md docs/unit_tests.md
-          rm -rf out.md
-          python3 .github/generate_inline_svg_md.py trace_scap.svg
-          mv out.md docs/scap_file.md
-          rm -rf out.md    
-          
+          mv trace_tests.svg docs/trace_tests.svg
+          echo '<object data="../trace_tests.svg" type="image/svg+xml" id="unit_tests"></object>' > docs/unit_tests.md
```
Comment (FedeDP) [inline .github/workflows/pages.yml:104]: Using the object tag, we have an interactive svg!

**Comment by FedeDP** [comment]
/unhold
this is ready!

**Review by LucaGuerra — APPROVED** [review]

**Review by Andreagit97 — APPROVED** [review]
/approve

**Pull request merged**